src/js/sdk.tsx

```diff
@@ -44,6 +45,8 @@ export function init(passedOptions: ReactNativeOptions): void {
   }
 
   useEncodePolyfill();
+  enableSyncToNative(getGlobalScope());
+  enableSyncToNative(getIsolationScope());
```

Comment:
M: We only need to sync when native is enabled, when disabled we could use the original Scope

```suggestion
  if (enableNative) {
     enableSyncToNative(getGlobalScope());
     enableSyncToNative(getIsolationScope());
  }
```

Reply:
Yes, that's a good point.